Makefile

```diff
@@ -68,31 +49,34 @@ uninstall:
 .PHONY: reinstall
 reinstall: uninstall install
 
-install-ocamlformat:
-	opam install -y ocamlformat.$$(awk -F = '$$1 == "version" {print $$2}' .ocamlformat)
+OCAMLFORMAT_VERSION := $(shell opam show ./dune.opam --just-file |grep ocamlformat|cut -d'"' -f4 )
```

Comment:
I think you can use `--raw` and `-fdepends` here to select the right section of the OPAM file. Personally I'd probably use `grep` to get the version instead of `cut` since it feels a it roundabout to use `"` as delimiter but well, it does work.

Reply:
`--raw` is implied by `--just-file`. `-fdepends` will restrict a bit the input but it's still necessary to select the line that contains ocamlformat, so it will be one `grep` followed by one `grep -o`, is that what you had in mind?

Reply:
Indeed, but by specifying `-fdepends` you make sure that the ocamlformat mentioned is in the depends field and not e.g. in `conflicts` or something. It's not a big deal either way as we control the opam file for the most part.